Update Music Assistant with Bronze docs requirements (#38172)
home-assistant/home-assistant.github.io: source/_integrations/music_assistant.markdown

All of the Home Assistant [Media Player Control Actions](https://www.home-assistant.io/integrations/media_player/#media-control-actions) are supported.

The `media_content_id` payload for `media_player.play_media` can be any of the following:

> - The name of a track, artist or album. (for example, `Queen`)

Reviewer: ```suggestion
- The name of a track, artist, or album. For example, `Queen`.
```

> - A track or album combined with the artist name (for example, `Queen - Innuendo`)

Reviewer: ```suggestion
- A track or album combined with the artist's name. For example, `Queen - Innuendo`.
```

> - A streaming provider URI (for example, `spotify://artist/12345`)

Reviewer: ```suggestion
- A streaming provider URI. For example, `spotify://artist/12345`.
```

The `media_content_id` payload for `media_player.browse_media` must be a URI of the form `library://artist/1`, `library://album/20`, or `spotify://album/5zj4Ej0FrlJQaSo0d6cttH`. The type of item that the URI refers to must be an album or artist.

> These URIs can be obtained from, for example, the output of the `get_library` or `search` actions described below or the `media_player.browse_media` HA action. 

Reviewer: ```suggestion
These URIs can be obtained from, for example, the output of the `get_library` or `search` actions described below or the `media_player.browse_media` action from Home Assistant. 
```

{% include integrations/config_flow.md %}

### Manual configuration

> Under normal circumstances, Home Assistant automatically discovers your running Music Assistant Server. If there is something special about the HA or MA setup (for example, the MA server is running as a remote Docker container) or discovery is not working, you can manually specify the URL to your Music Assistant server. If the Music Assistant Server is not installed, then follow these [installation instructions](https://music-assistant.io/installation/).

Reviewer: ```suggestion
Under normal circumstances, Home Assistant automatically discovers your running Music Assistant Server. If there is something special about the Home Assistant or MA setup (for example, the MA server is running as a remote Docker container) or discovery is not working, you can manually specify the URL to your Music Assistant server. If the Music Assistant Server is not installed, then follow these [installation instructions](https://music-assistant.io/installation/).
```

> ## Removing the integration

Reviewer: I would move the removal instructions to the end of the page, as it marks the end of the integration's life-cycle.

Author: Thanks done!

This integration follows standard integration removal. No extra steps are required.

{% include integrations/remove_device_service.md %}

> After deleting the integration, go to the HA SETTINGS >> ADD-ONS and remove the Music Assistant Add-on from there as well (if installed).

Reviewer: ```suggestion
After deleting the integration, go to the {% my supervisor_addon title="**Settings** > **Add-ons**" addon="core_samba" %} and remove the **Music Assistant** add-on from there as well (if installed).
```

media_content_id: 'The Best of Disco'
```

> ### Action `media_player.browse_media`

Reviewer 2: This is a media player action, not specific to music assistant.

It thus isn't integration specific and thus shouldn't be documented here but in the media player documentation.

Author: But the media player documentation says that the media_content_id is “integration specific” so this section is intended to show the specifics?

Reviewer 3: browse_media is a (global) action of the media_player platform and not specific to MA, so it shouldn't be documented here, it's already documented in the media_player docs.

If you want to have something in the documentation about the supported media content type's and media content id(s), that would maybe better be fit in a notes section ?

Or maybe some specific section where it says you can use all the default media player actions from HA and when then describe the fact that MA supports the media_content_id in uri format.

Author: All text pertaining to HA actions removed. media_content_id info added to the top.

## Notes

- Any Home Assistant players added to Music Assistant will appear duplicated as the MA version of the player is created. The original HA player can be hidden if desired.

## Removing the integration

> This integration follows standard integration removal. No extra steps are required.

Reviewer: ```suggestion
This integration follows standard integration removal.
```
